src/rules/padding-before-test-blocks.ts

```diff
@@ -5,7 +5,10 @@ export default {
   create(context) {
     const ctx = Object.create(context, {
       options: {
-        value: [{blankLine: 'always', prev: '*', next: 'test'}]
+        value: [
+          {blankLine: 'always', prev: '*', next: 'test'},
+          {blankLine: 'always', prev: '*', next: 'it'},
```

Comment:
I added this in to treat `it` the same as `test`. It looks like this is how the original rule worked and it makes sense to me.